contentgrid-spring-data-rest/src/testFixtures/java/com/contentgrid/spring/test/fixture/invoicing/model/Invoice.java

```diff
@@ -0,0 +1,55 @@
+package com.contentgrid.spring.test.fixture.invoicing.model;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class Invoice {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private UUID id;
+
+    private String number;
+
+    private boolean draft;
+
+    private boolean paid;
+
+    @ManyToOne
```

Comment:
I see a `@ManyToOne` here, but I am missing a test for `@OneToOne`.

I know that it "should" be the same and hit the same codepath in the controller, but we still need a test for that functionality.

Reply:
`Order` has now a `@OneToOne` to `ShippingAddress`